federated-etl.md

```diff
@@ -107,3 +112,9 @@ prometheus:
     * To verify the entire pipeline is working, either query `Allocations/Assets` or view the respective views on the frontend. Multi-cluster data should appear after:
         * The Federator has run at least once.
         * There was data in the Federated Storage for the Federator to have combined.
+
+### Data Recovery
+
+When using ETL Federation, there are several methods to recover Kubecost data in the invent of data loss:
+
+The [backup and alerting](https://docs.kubecost.com/install-and-configure/install/multi-cluster/federated-etl/federated-etl-backups-alerting) has detail regarding the various methods.
```

Comment:
Can we adjust the link here to instead show

```
[backups and alerting](/federated-etl-backups-alerting.md)
```

This way, if you're reading the doc on Github you get redirected within Github. If you're reading on Gitbook, you get redirected within Gitbook